src/components/UnlinkedTask.js

```diff
@@ -47,23 +47,25 @@ const UnlinkedTask = (props) => {
 const styles = EStyleSheet.create({
   container: {
     alignSelf: 'flex-start',
-    margin: 20
+    margin: 15,
+    position: 'absolute',
```

Comment:
isn't absolute positioning a bad idea on different view ports?

Reply:
Yes, I think you are right on the absolute positioning. I arrived at this solution because there appears to be some wonkiness with where the unlinked tasks render depending on the image size that I couldn't quite nail down while hacking away yesterday. Unfortunately, a fix is needed by Monday to get an Apple review and release out in time.

I will double check the rendering on several more device screen sizes. iPhone SE/6 and Nexus 5 appeared well according to this PR.

Reply:
Layout appears to be good on iPhones 5-7 and various size Nexus screens.

Reply:
If you are happy with it and it's needed then we can revisit it once it's out.